azext_edge/edge/params.py

```diff
@@ -137,7 +137,7 @@ def load_iotops_arguments(self, _):
         context.argument(
             "ops_service",
             options_list=["--ops-service", "--svc"],
-            choices=CaseInsensitiveList(["akri", "dataprocessor", "deviceregistry", "mq", "opcua"]),
+            choices=CaseInsensitiveList(["akri", "dataprocessor", "deviceregistry", "broker", "opcua"]),
```

Comment:
honestly would like to see this baked into the enum somehow (maybe another function like list to return the ops_service list)